### PR TITLE
Toogle company admin and employee

### LINKED ISSUE #17: Company admin can edit rights for employes
Currently we have application admin which can change roles on global level.
Company admins should be able to list only users which are employed in that company, and change rights for them. Here we should have employee, and  company_admin.

From this page company admin can fire employee, and restrict him all rights.

### PR DESCRIPTION
**Pull request by KenzR**
Closes #17

Company admin now can toogle employees roles and fire employees

### CODE REVIEW

**Comment by IvanPavlovic92** [comment]
Company admin should have link in menu for managing employees.

Company admin should be able to invite person who is not company employee to be company admin.
(should be separate issue)

Company admin should not be able to change his role to employee if he is only company admin.

File: app/controllers/company_admin_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class CompanyAdminController < ApplicationController
```
Comment (sls-lnluksa) [inline app/controllers/company_admin_controller.rb]: We don't need this file.
All logic should be in company_controller and access should be controlled in company_policy.
Remove this file and move logic to company.

File: app/controllers/company_admin_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    redirect_to root_path, alert: "You have no rights for requested page" if current_user.current_company_rights.employee? or !(current_user.active_companies.include? current_user.current_company)
+    if current_user.current_company_rights.company_admin? and current_user.active_companies.include? current_user.current_company
+      @user_ids = UserCompany.where(company_id: current_user.current_company.id, active: true).pluck(:user_id)
+      @users = User.where(id: @user_ids)
+    end
+
+  end
+
+  def update
+    redirect_to root_path, alert: "You have no rights for requested page" if current_user.current_company_rights.employee? or !(current_user.active_companies.include? current_user.current_company)
```
Comment (sls-lnluksa) [inline app/controllers/company_admin_controller.rb]: Try to write all of this in policy file.

File: config/locales/nn.yml [hunk 3]
```diff
@@ -53,6 +53,7 @@ nn:
   forms:
     submits:
       change_role: "Endre rolle"
+      change_employee_role: "Endre rolle"
```
Comment (sls-lnluksa) [inline config/locales/nn.yml]: If you have some translations that are used in multiple pages, make it global or use same translation. Do not create duplicates.

**Pull request merged**